MaybeGame/GameMaster.cs

```diff
@@ -3,69 +3,92 @@
 
 namespace MaybeGame
 {
-    partial class MainProgram
+    public class GameMaster
```

Comment:
```suggestion
    public class GameManager
```
GameMaster could be Admin type later on

Reply:
That is true